apps/consent/app/login/phone/form.tsx

```diff
@@ -0,0 +1,118 @@
+"use client";
```

Comment:
it's not super clear to me as to why we have a `phone` subfolder but not an `email` subfolder given those are the 2 ways to authenticate.

Reply:
we can create separate folder for email, but it will require one more redirect.

Reply:
is this the right name then?

Reply:
of the folder?